<launch>
  <node name="$(env ROS_HOSTNAME)_usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="320" /> <!-- Default Value for camera -->
    <param name="image_height" value="240" /> <!-- Default Value for camera -->
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
</launch>
